src/main/scala/scala/tools/refactoring/implementations/oimports/Region.scala

```diff
@@ -70,8 +84,19 @@ object MiscTools {
   def isScalaLanguageImport[G <: Global](g: G): g.Import => Boolean = {
     import g._
     val language = newTermName("language")
+
+    @tailrec
+    def isScalaLanguage(qual: Select): Boolean = qual match {
```

Comment:
discussed, merge `CompilationUnitDependencies` with new approach should purge redundant implementation